### DISCUSSION TITLE
Home Assistant Support

### DISCUSSION THREAD

**Discussion by elpeterson** [awawa-dev/HyperHDR#205] [Ideas]
Hyperion.NG has Home Assistant support through the original Hyperion Integration [here](https://www.home-assistant.io/integrations/hyperion/). There appears to be an authorization error when trying to setup HyperHDR through the same integration. Is this something you would be willing to add support for? Being able to send commands to switch lighting modes when I send ADB commands to my TV through Home Assistant (ie music light when Spotify is launched) would be huge

**Comment by awawa-dev** [comment]
Hi
HyperHDR is not compatible with HA Hyperion.NG plugin. Please use dedicated HA HyperHDR integration @[mjoshd](https://github.com/mjoshd) https://github.com/mjoshd/hyperhdr-ha

**Reply by elpeterson** [reply]
Oh hells yes. Thank you! I had looked at a past "Idea" and saw that since it was a derivative of Hyperion that it was compatible. This is perfect, thank you so much.
